packages/client/wallets/aa/src/utils/user.ts

```diff
@@ -0,0 +1,75 @@
+import { UserIdentifier, UserIdentifierParams } from "@/types/Config";
```

Comment:
are these copied from somewhere?

Reply:
I think we're copying these from crossbit-main. If that's the case, I'd love to put these into `@crossmint/client-sdk-base` and let both the AA sdk and Crossbit-main pull them in as dependencies.
Is that possible?

Reply:
This change should be at another PR, if not we cannot use it because it is not deployed and will break everything. I'm going to create a new PR for that and after that we can switch to use `@crossmint/client-sdk-base`